Dockerfile

```diff
@@ -42,18 +39,18 @@ COPY examples /examples
 
 WORKDIR /tmp
 RUN apk add --no-cache npm make git jq ca-certificates openssl unzip wget && \
-    wget "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip" && \
+    wget "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_${TARGETARCH}.zip" && \
     unzip "terraform_${TERRAFORM_VERSION}_linux_amd64.zip" -d /usr/local/bin
 
-RUN wget -O /usr/local/bin/kustomize "https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_amd64" && \
+RUN wget -O /usr/local/bin/kustomize "https://github.com/kubernetes-sigs/kustomize/releases/download/v${KUSTOMIZE_VERSION}/kustomize_${KUSTOMIZE_VERSION}_linux_${TARGETARCH}" && \
     chmod +x /usr/local/bin/kustomize
 
 RUN go install cuelang.org/go/cmd/cue@latest
 
 WORKDIR /examples
 
 ## RELEASE ##
-FROM alpine:3.17.0
+FROM alpine
```

Comment:
Please revert this change, we prefer to version all dependencies including container base images.

Reply:
> Please revert this change, we prefer to version all dependencies including container base images.

@jalseth updated